Makefile

```diff
@@ -686,6 +686,10 @@ ALL_VERIFY_CHECKS = licenses boilerplate shellcheck tiltfile modules gen convers
 .PHONY: verify
 verify: $(addprefix verify-,$(ALL_VERIFY_CHECKS)) lint-dockerfiles ## Run all verify-* targets
 
+.PHONY: verify-go-directive
+verify-go-directive:
+	TRACE=$(TRACE) ./hack/verify-go-directive.sh -g 1.21
```

Comment:
Could we try to parameterize this? there's a `GO_VERSION` variable in this file

Reply:
Not really because the GO_VERSION in the makefile and the one in the go directive are not always in sync (e.g. on release branches)